css/style.css

```diff
@@ -1,3 +1,1033 @@
+    position: absolute;
+    display: inline-block;
+    color: white;
+    font-size: 20px;
+    text-transform: uppercase;
+    text-align: justify;
+    white-space: pre-line;
+    left: 30px;
+    top: 40px;
+    left: 20px;
+    line-height: 1.2;
+  }
+
+  .goto__shop {
+    font-size: 12px;
+  }
+
+  .goto__arrow {
+    width: 45.51px;
+    cursor: pointer;
+  }
+
+  .products__title {
+    font-size: 20px;
+    margin: 0 auto;
+    text-align: center;
+    text-transform: uppercase;
+  }
+
+  .products__list {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 248px;
+    justify-content: center;
+    margin: 0 auto;
+    text-align: center;
+    position: relative;
+  }
+
+  /* rasta */
+
+  .products__product-rasta {
+    align-self: flex-start;
+    text-align: left;
+    position: relative;
+    width: 280px;
+    height: 340px;
+  }
+
+  .product-rasta_r {
+    position: absolute;
+    display: inline-block;
+    right: 60px;
+    bottom: 120px;
+    background: #b62e3c;
+    border-radius: 50%;
+    width: 15px;
+    height: 15px;
+  }
+
+  .product-rasta_b {
+    position: absolute;
+    display: inline-block;
+    right: 40px;
+    bottom: 120px;
+    background: #0e1211;
+    border-radius: 50%;
+    width: 15px;
+    height: 15px;
+  }
+
+  .product-rasta_g {
+    position: absolute;
+    display: inline-block;
+    right: 20px;
+    bottom: 120px;
+    background: #c4c4c4;
+    border-radius: 50%;
+    width: 15px;
+    height: 15px;
+  }
+
+  .product-rasta_y {
+    position: absolute;
+    display: inline-block;
+    right: 0;
+    bottom: 120px;
+    background: #fdb431;
+    border-radius: 50%;
+    width: 15px;
+    height: 15px;
+  }
+
+  .products__product-camel {
+    display: none;
+  }
+
+  .products__product-mandel {
+    display: none;
+  }
+
+  .products__product-cloud {
+    display: none;
+  }
+
+  .products__img {
+    display: block;
+    width: 165.65px;
+    height: 165.42px;
+    margin: 0 auto 15px auto;
+  }
+
+  .products__name {
+    position: absolute;
+    font-size: 15px;
+    line-height: 14px;
+    top: 165px;
+    left: 14px;
+  }
+
+  .products__cost {
+    position: absolute;
+    font-size: 20px;
+    line-height: 19px;
+    font-weight: bold;
+    top: 200px;
+    left: 14px;
+  }
+}
+
+/* mobile 375 */
+
+@media all and (min-width: 375px) {
+  .page {
+    max-width: 375px;
+  }
+
+  .page__header {
+    max-width: 375px;
+  }
+
+  .header__logo-mobile {
+    left: 45px;
+  }
+
+  .menu__burger {
+    right: 45px;
+  }
+
+  .new-collection__goto .goto__title {
+    top: 48px;
+    left: 45px;
+    line-height: 19px;
+  }
+
+  .goto__shop {
+    position: absolute;
+    top: 65px;
+  }
+
+  .goto__arrow {
+    width: 45.51px;
+    cursor: pointer;
+  }
+
+}
+
+/* tablet 768 */
+
+@media screen and (min-width: 768px) {
+  .page {
+    max-width: 768px;
+  }
+
+  .page__header {
+    max-width: 768px;
+    height: 113.54px;
+    margin: 0 auto; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)